tests/validation/test_ou3_correlated_kalman_gain.py

```
@@ -0,0 +1,72 @@
+import sys
+import unittest
+
+ROOT = Path(__file__).resolve().parents[2]
+sys.path.insert(0, str(ROOT / "tools"))
+
+from ou3_interval import Interval, matrix_add, matrix_mul, matrix_transpose
+from ou3_interval_linear_algebra import matrix_inverse_gauss_jordan
+import ou3_correlated_kalman_gain as CG
+
+
+def I(x):
```

Ruff (0.16.3)

[error] Ambiguous function name: `I`

(E743)